submissions/TaniaBondarenko/frogger_game/app.js

```diff
@@ -0,0 +1,117 @@
+const borders = {
+    rightBorder: 400,
+    leftBorder: 5,
+    upBorder: 410,
+    downBorder: 70
+};
+
+const Enemy = function (x, y) {
+    this.sprite = 'images/enemy-bug.png';
+    this.x = x;
+    this.y = y;
+    this.speed = 75;
+
+    this.render = function () {
+        ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+    };
+};
+
+    Enemy.prototype.update = function (dt) {
+        this.x += this.speed * dt;
+        if (this.x >= 500) {
```

Comment:
magic number.
Is this '500' the width of the game field?

Reply:
No, my width is 400, and I need 500 to make enemies-bugs moving smoothly.

Reply:
well this '500' is the magic number too...